code/modules/shuttle/shuttle.dm

```diff
@@ -246,6 +246,27 @@
 		reserved_area = null
 	return ..()
 
+/obj/docking_port/stationary/picked
+	var/list/shuttlekeys
```

Comment:
This comment (or lack thereof) does not meet our [documentation standards](https://github.com/tgstation/tgstation/blob/master/.github/AUTODOC_GUIDE.md#documenting-a-variabledefine), please update this.